database/elastic_mapping/elasticMapFactory.ts

```diff
@@ -31,10 +31,14 @@ export default {
             (property.type === 'select' || property.type === 'multiselect')
           ) {
             map.properties.suggestedMetadata.properties[property.name] = {
-              properties: propertyMappings[property.type](),
+              properties: {
+                ...propertyMappings[property.type](),
+                suggestion_confidence: {
```

Comment:
Not a big problem, but if we add suggestion_confidence here, we are adding that field to every property, but we only support text, numeric, and date.
- Pros: It's prepared for every new property we support in the future.
- Cons: We are adding fields that may never be used, and I know we reach the limit in the past.

Reply:
You are right, didn't  see that is inside of a conditional for only select and multiselect